Nomination pool members cannot participate in OpenGov (#4924)
w3f/polkadot-wiki: docs/learn/learn-nomination-pools.md

**For Ledger users:** Joining a nomination pool is possible only with the XL version of the Polkadot
Ledger App. This should be installed by default on Ledger Nano X and S Plus, but not on the Nano S.

> **If you become a nomination pool member, you cannot participate in Governance with bonded tokens,
> nor does the pool admin as funds are held in a

Reviewer: ```suggestion
**If you become a nomination pool member or a pool admin, you cannot participate in Governance with the bonded tokens in the pool, as they are held in a
```
